subjects/zappy/README.md

```diff
@@ -162,52 +165,52 @@ The time unit is defined by the function 1/t :
 
 - t is passed as parameter at the start of the server.
 - A time unit lasts 1/t seconds.
-
-- There cannot be any blocking when the clients are stopped, or in any phase of the game.
 - The Minerthians have second as an absolute time unit.
+- The formula for the execution of an action is `action/t`.
 - `t` is an integer. If `t=1` "advance" takes 7 seconds. We choose by default, `t=100` then "advance" will take 1/100 seconds.
 
-### Objects denomination
+### Objects Management
+
+It is impossible to distinguish two objects of the same class,. Only the class of an object can be identifiable, that is, two 'ambers' will have the same denomination, as they belong to the same class, but `food` will have a different class than the `stones`, that way you are able to identify the resources.
 
-It is impossible to distinguish two objects of the same class, only the classification of an object can be identifiable,
-that is, two 'ambers' will have the same denomination, as they belong to the same class.
+### Teams/Families
 
-### Teams
+The purpose is to create a `team/family` of six players and get them to level 8!
 
-At the beginning a team is made of `n` player and only `n`. Each player is controlled by a client. The clients cannot communicate or exchange amongst each other data outside of the game, in any way.
+At the beginning of the game your family only has one member, so you have to complete the action of `calling` in order to get more family members.
 
-At the beginning the client has 10 food units, he can therefore survive 1260 time units, 1260/t seconds.
+Each player starts with 10 food and is controlled by a `client`. The `clients` cannot communicate or exchange amongst each other data outside of the game, in any way.
 
-### Reproduction
+### Calling
 
-Executing the `fork` command results in the production of an `egg`, which allows for reproduction.
+Execution of the `fork` command results in the invocation of a `ship`, which allows replay.
 
-Once the egg is laid, the player who laid the egg can go to it and hatch it, when it hatches, a new player will be born and given a random direction.
-This operation authorizes the connection of a new client.
+Once the `ship` arrives on the planet, the player who called it can go to it and start the process of opening doors to receive a new member, when the process is finished, a new player will appear and receive a random direction.
```

Comment:
This changes the dynamics of the game in a dangerous way. With the eggs the player already playing spends 42 units of time making the egg... then the player can continue to go on with his life and at one point the egg opens and a new client join.

Here you say "once the ship arrives the player opens the doors" which is the opposite in terms of when the time is spent and also way more complex for the client.
Let's just say that the command is "call_a_friend" and the phone call takes 48t or something like that.
Then the friend will arrive in that position after 400ish t and that will be a new client joining the game. (the one that called the friend doesn't have anything else to do with the process at the arrival of the friend and can be where he wants)

Reply:
done